config/prismjs/dvc-commands.js

```diff
@@ -42,6 +42,8 @@ module.exports = [
   'gc',
   'freeze',
   'fetch',
+  'exp',
+  'experiments',
   'doctor',
```

Comment:
Unrelated to this PR but it looks like just adding these is enough for the engine to also link subcommands?

![image](https://user-images.githubusercontent.com/1477535/108145019-5e063a00-7090-11eb-95e2-dc1bda0de86f.png)
> https://dvc-org-ref-experiments-dv5pbs.herokuapp.com/doc/user-guide/experiment-management

So should we prune subcommands from that list, @rogermparent ? Thanks

Reply:
Considering the links have the subcommands as fragments, this seems like an intended behavior that just happens to be taken to an extreme here because `dvc exp` uses a lot of subcommands.

Since the current behavior is at worst slightly busy and possibly even useful, the most I'd give to the issue of skipping subcommands is a lower-priority discussion ticket on the rules for auto-links. That logic is a little crufty as it is, so getting some input on a future refactor that unifies that little nest of code could be useful.

Reply:
OK. Reminds me of #1614 BTW. Maybe we can add feedback in there? Resolving this for now. Thanks